translate/rus.json

```diff
@@ -71,7 +71,7 @@
 "bottom":"дно",
 "brackets":"древо",
 "brown":"коричневый",
-"busted openings":"забитые дебюты",
+"busted openings":"победа и поражение",
```

Comment:
I think I would uniform the stats description and instead of "busted openings" I would call it as "Win & Loss" (don't forget to change it in every language!) What do you think?

Reply:
I thought about it but people prefer to use the "busted" terms, and they also prefer to use "reverse kills" instead of "double wins", btw.

Reply:
Yeah, but "забитые дебюты" is 100% wrong translation. I will try to come up with something better in the future. Not now though. I leave this PR open, until I will come up with a good translation for this.